ansible/roles/dev/files/django.bashrc

```diff
@@ -0,0 +1 @@
+export DJANGO_SETTINGS_MODULE=pulpcore.app.settings
```

Comment:
I am pretty sure that the `environment` keyword only works within Ansible scope. If we want to use Django from the shell this variable is not required because the user can also pass their own location.